### DISCUSSION TITLE
BlenderGIS elevation height differs between computers

### DISCUSSION THREAD

**Discussion by bobg3723** [domlysz/BlenderGIS#485] [General]
I created a map with BlenderGIS with srtm30 height data on an i3 processor system with 16 GB system ram, integrated GPU
When I tried creating the same map, same data on an older system with an AMD 8150  with 8 GB ram, 8 GB pci-e graphics card, the map's height data was not a detailed in heights (mountain, and peak were smoothed out and not as bumpy) in the 3D view on either perspective or orthographic. It's as if the mesh nodes were not as dense. The map is 45 km wide by 85 km long.

When the BLEND file converts to a 200k DXF file. It's big. I'm more interested in fast rendering instead of pretty rendering.  Is there a setting in Blender I can adjust on the AMD system to compensate for this loss in height resolution?

Or am I just lacking in computer ram on the older system?

**Comment by domlysz** [comment]
I suspect the extents of your 2 tests are not exactly the same. Larger area need more vertices : try to increase the subsurf level in  the modifier stack or add another one. 
By the way 45x85km is probably a bit large for Blender especially if you expect a good resolution.

**Reply by bobg3723** [reply]
I went back  and tried to repeat the problem on the original i3 computer and the problem is also there, now. Here's a picture of the original peninsula I converted to DXF and the a repeat on same computer a few days later.
![SRTM30 no longer fine resolution](https://user-images.githubusercontent.com/83843981/117589868-ee817300-b0f1-11eb-8a7f-c0239b29c239.jpg)

**Reply by bobg3723** [reply]
PROBLEM SOLVED!

Your suggestion to subdivide the mesh worked. My problem was that I forgot to subdivide the mesh prior to getting the SRTM30 elevation from the server.  It works perfectly on both computers.  The x and y dimensions are 96km by 166km. 

I did discover one more problem I didn't mention, but I solved that by scaling.  It seems that the x and y scale of the map is different from Google Earth and it's by a significant percentage. But that's a non-issue if I re-scaled the map.

**Reply by domlysz** [reply]
Take a look at the FAQ about scale distorsions :
https://github.com/domlysz/BlenderGIS/wiki/FAQ#why-my-measurements-on-basemaps-reference-seems-wrong-

**Comment by bobg3723** [comment]
Dear Sir,

I learned something today. Thank you, domlysz.

**Comment by bobg3723** [comment]
I get it about distortion. If the latitude is a point in the formula for correct distance X = COS(Latitude), how do I arrive at the correct measure of a Tape Measure that spans over multiple latitudes or longitudes or diagonally across both?

**Comment by domlysz** [comment]
There are some complex formulas to compute the distance between 2 angular points on the ellipsoid, but it's really not what you want.
If you need good measurements then you should use an adaptated projection system that minimize the distorsions on your area of interest. That's what projections are for.

**Comment by bobg3723** [comment]
The projection coordinate system I'm going to re-project to is (EPSG:27700) for the U.K. I'll learn how to do that with QGIS and when I've understood that process I'll try and use the data for that projection to bring into BlenderGIS. I'll have to study up on that process so I can be in a position to ask you pertinent questions. I just don't know enough to know what to ask, so until then thank you for your suggestions. Please feel free to suggest anything I may not have considered.
